#! /bin/sh /usr/share/dpatch/dpatch-run
## manpage.dpatch by Samuel Mimram <smimram@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove some invalid characters.

@DPATCH@
diff -urNad menhir-20060615~/menhir.1 menhir-20060615/menhir.1
--- menhir-20060615~/menhir.1	2006-06-15 07:26:06.000000000 +0000
+++ menhir-20060615/menhir.1	2006-06-24 19:25:19.000000000 +0000
@@ -119,7 +119,7 @@
 
 .SH AUTHOR
 .B menhir
-was written by François Pottier and Yann Régis-Gianas.
+was written by Fran\(,cois Pottier and Yann R\('egis-Gianas.
 .PP
 This manual page was written by Samuel Mimram <smimram@debian.org>,
 for the Debian project (but may be used by others).
